content/en/blog/2024/state-profiling.md

```diff
@@ -0,0 +1,117 @@
+---
+title: The State of Profiling
+linkTitle: Profiling state
+date: 2024-10-24
```

Comment:
question on timing: we had the GC election announcement blog post today, so maybe we publish this one tomorrow?

```suggestion
date: 2024-10-25
```